client/components/app.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
```

Comment:
Looks good. Would consider destructuring and importing required modules. Not required.